<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="med000008316" failures="1" tests="13" name="died_procesos_generales.pgForgotPassTest.ForgotPassTest" time="135.853" errors="0" timestamp="20 feb. 2019 19:34:00 GMT+00:00" skipped="1">
  <testcase classname="died_procesos_generales.pgForgotPassTest.ForgotPassTest" name="ForgotPassCaseTest01" time="45.525"/>
  <testcase classname="died_procesos_generales.pgForgotPassTest.ForgotPassTest" name="ForgotPassCaseTest02" time="9.275">
    <skipped/>
  </testcase> <!-- ForgotPassCaseTest02 -->
  <testcase classname="died_procesos_generales.pgForgotPassTest.ForgotPassTest" name="ForgotPassCaseTest02" time="9.016">
    <failure message="expected [Por favor revisar el código de validación enviado a su celular, puede que tarde unos minutos en llegarle.] but found [Usted tiene una petición generada, espere 2 minutos e intente de nuevo.]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Por favor revisar el código de validación enviado a su celular, puede que tarde unos minutos en llegarle.] but found [Usted tiene una petición generada, espere 2 minutos e intente de nuevo.]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at died_procesos_generales.pgForgotPassPage.ForgotPassPage.verifyMessage01_aroundBody18(ForgotPassPage.java:247)
	at died_procesos_generales.pgForgotPassPage.ForgotPassPage$AjcClosure19.run(ForgotPassPage.java:1)
	at org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149)
	at io.qameta.allure.aspects.StepsAspects.step(StepsAspects.java:46)
	at died_procesos_generales.pgForgotPassPage.ForgotPassPage.verifyMessage01(ForgotPassPage.java:247)
	at died_procesos_generales.pgForgotPassTest.ForgotPassTest.ForgotPassCaseTest02(ForgotPassTest.java:88)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- ForgotPassCaseTest02 -->
  <testcase classname="died_procesos_generales.pgForgotPassTest.ForgotPassTest" name="ForgotPassCaseTest03" time="5.046"/>
  <testcase classname="died_procesos_generales.pgForgotPassTest.ForgotPassTest" name="ForgotPassCaseTest04" time="6.581"/>
  <testcase classname="died_procesos_generales.pgForgotPassTest.ForgotPassTest" name="ForgotPassCaseTest05" time="7.510"/>
  <testcase classname="died_procesos_generales.pgForgotPassTest.ForgotPassTest" name="ForgotPassCaseTest06" time="6.013"/>
  <testcase classname="died_procesos_generales.pgForgotPassTest.ForgotPassTest" name="ForgotPassCaseTest07" time="5.697"/>
  <testcase classname="died_procesos_generales.pgForgotPassTest.ForgotPassTest" name="ForgotPassCaseTest08" time="8.717"/>
  <testcase classname="died_procesos_generales.pgForgotPassTest.ForgotPassTest" name="ForgotPassCaseTest09" time="7.000"/>
  <testcase classname="died_procesos_generales.pgForgotPassTest.ForgotPassTest" name="ForgotPassCaseTest10" time="7.031"/>
  <testcase classname="died_procesos_generales.pgForgotPassTest.ForgotPassTest" name="ForgotPassCaseTest11" time="10.736"/>
  <testcase classname="died_procesos_generales.pgForgotPassTest.ForgotPassTest" name="ForgotPassCaseTest12" time="7.706"/>
</testsuite> <!-- died_procesos_generales.pgForgotPassTest.ForgotPassTest -->
